torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -333,17 +345,3 @@ std::vector<at::Tensor> CreateXlaTensors(
 
 }  // namespace bridge
 }  // namespace torch_xla
-
-namespace torch {
-namespace lazy {
-
-torch_xla::XLATensor GetXlaTensorOrCreateForWrappedNumber(
```

Comment:
took me a minute but i think these changes are just moving namespace and cosmetic changes to the same logic

Reply:
yea, also I make it return `XLATensorPtr` instead of `XLATensor`